.github/workflows/discussion-triage.yml

```
@@ -0,0 +1,28 @@
+name: Discussion Triage
+on:
+  issues:
+    types:
+      - labeled
+jobs:
+  add-comment:
+    if: github.event.label.name == 'discuss'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create a discussion in github/cli
```

In general, avoiding hard coding repository details helps make long term use a little easier:

```suggestion
      - name: Create a discussion in ${{ github.repository }}
```

---

Is this correct? The action runs in `cli/cli` but the discussion gets created in `github/cli`.

---

I definitely missed that, thanks.